examples/ruby/create_themes.rb

```diff
@@ -0,0 +1,53 @@
+require 'looker-sdk'
+
+# get API creds from environment variables
+sdk = LookerSDK::Client.new(
+  :client_id => ENV['LOOKERSDK_CLIENT_ID'],
+  :client_secret => ENV['LOOKERSDK_CLIENT_SECRET'],
+  :api_endpoint => ENV['LOOKERSDK_BASE_URL']
+)
+​
+# NOTE: Custom themes needs to be enabled by Looker. 
+# Unless custom themes are enabled, only the automatically generated default theme can be used. 
+# Please contact your Account Manager or support@looker.com to update your license for this feature.
+
+# Example formatting for the theme details to be created
+#  example_custom_theme = {
+# 	 	:name=> "my_custom_theme",
+# 	  	:settings=> {
+# 	    	:background_color=> "#d8e2f2",
+# 		    :base_font_size=> "12px",
+# 		    :color_collection_id=> "",
+# 		    :font_color=> "#143f85",
+# 		    :font_family=> "Arial",
+# 		    :font_source=> "",
+# 		    :info_button_color=> "#0087e1",
+# 		    :primary_button_color=> "#64518a",
+# 		    :show_filters_bar=> true,
+# 		    :show_title=> true,
+# 		    :text_tile_text_color=> "",
+# 		    :tile_background_color=> "#efaa3b",
+# 		    :tile_text_color=> "#3a4245",
+# 		    :title_color=> "#7d0f48",
+# 		    :warn_button_color=> "#980c11"
+# 	  }
+# 	}
+​
+# these are general examples and not necessarily themes that follow the brand guidelines for these companies
+airbnb_like = {:name=>"airbnb", :settings=>{:background_color=>"#faf3f2", :base_font_size=>"12px", :color_collection_id=>"5f313589-67ce-44ba-b084-ec5107a7bb7e", :font_color=>"#143f85", :font_family=>"Arial", :font_source=>"", :info_button_color=>"#0087e1", :primary_button_color=>"#FF5A5F", :show_filters_bar=>true, :show_title=>true, :text_tile_text_color=>"", :tile_background_color=>"#faf3f2", :tile_text_color=>"#FF5A5F", :title_color=>"#FF5A5F", :warn_button_color=>"#980c11"}}
```

Comment:
sorry, late comment here but consider reformatting these for a/ new-style symbol keys and b/ vertical readability:

```ruby
airbnb_like = {
  name: "airbnb",
  settings: {
    background_color: "#faf3f2",
    base_font_size: "12px",
    color_collection_id: "5f313589-67ce-44ba-b084-ec5107a7bb7e",
    font_color: "#143f85",
    font_family: "Arial",
    font_source: "",
    info_button_color: "#0087e1",
    primary_button_color: "#FF5A5F",
    show_filters_bar: true,
    show_title: true,
    text_tile_text_color: "",
    tile_background_color: "#faf3f2",
    tile_text_color: "#FF5A5F",
    title_color: "#FF5A5F",
    warn_button_color: "#980c11",
  },
}
```